examples/virtual-device-2-eia-connection/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.5.2"
+  required_providers {
+    equinix = {
+      source  = "equinix/equinix"
+      version = ">= 1.34.0"
```

Comment:
Terraform release including the EIA change is 1.36.0. Should update this to be `">= 1.36.0"`